parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -161,9 +161,9 @@ pub mod pallet {
 		}
 	}
 
-	impl<T: Config> Pallet<T> {
+	impl<T: Config> SubmitMessage for Pallet<T> {
 		/// Submit message on the outbound channel
-		pub fn submit(origin: &ParaId, handler: u16, payload: &[u8]) -> DispatchResult {
+		fn submit(origin: &ParaId, handler: u16, payload: &[u8]) -> DispatchResult {
```

Comment:
`ParaId` is really just struct with a single `u32` field. Surely it implements `Copy`, in which case there is no need to pass by reference

Reply:
addressed in [da6a3e0](https://github.com/Snowfork/snowbridge/pull/805/commits/da6a3e0ed03ef96c989bc8a54d1383afa2523286)